src/createServer.js

```diff
@@ -1,3 +1,61 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = [];
+    const input = req.url.split('?');
+    const params = new URLSearchParams(input[1]);
+    const toCase = params.get('toCase');
+    const data = input[0].substring(1);
```

Comment:
The code assumes that the URL path will always contain a text to convert. If the path is empty, `data` will be an empty string, which is already checked later, but it might be better to handle this earlier or ensure the path is correctly parsed.